src/test/java/uk/gov/companieshouse/insolvency/delta/mapper/CaseMapperTest.java

```diff
@@ -26,7 +26,7 @@
 class CaseMapperTest {
 
     private static final String companyNumber = "1232466";
-    private static final String mortgageId = "1232466";
+    private static final String mortgageId = "1368018";
```

Comment:
This is a confirmed example from the db for which we have the expected encoding, handy for verifying the implementation is correct.